lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -437,7 +437,6 @@ def _section_course_info(course, access):
         section_data['grade_cutoffs'] = reduce(advance, sorted_cutoffs, "")[:-2]
     except Exception:  # pylint: disable=broad-except
         section_data['grade_cutoffs'] = "Not Available"
-    # section_data['offline_grades'] = offline_grades_available(course_key)
```

Comment:
This line was last edited 2.5 years ago in https://github.com/edx/edx-platform/commit/cd746bf8e5127c6049950ef002c5ca9e1aebed65. Being commented out all that time, I think we're pretty safe to delete it now.

https://github.com/edx/edx-platform/blame/master/lms/djangoapps/instructor/views/instructor_dashboard.py#L440